### PR TITLE
feat: allow customizing lua_package_path & lua_package_cpath

### PR DESCRIPTION
**Pull request by spacewander**
Signed-off-by: spacewander <spacewanderlzx@gmail.com>

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by dabue — APPROVED** [review]

File: .travis/apisix_cli_test/test_main.sh [hunk 1]
```diff
@@ -1013,3 +1013,27 @@ echo "passed: show password error successfully"
...
+
+rm logs/error.log
+make init
+make run
+
+sleep 0.5
+make stop
+
+if grep "failed to load plugin [3rd-party]" logs/error.log > /dev/null; then
+    echo "failed: 3rd party plugin can not be loaded"
```
Comment (Firstsawyou) [inline .travis/apisix_cli_test/test_main.sh:1036]: I think it’s better to use `3rd-party` to keep the plugin name consistent,

File: .travis/apisix_cli_test/test_main.sh [hunk 2]
```diff
@@ -1013,3 +1013,27 @@ echo "passed: show password error successfully"
...
+    exit 1
+fi
+echo "passed: 3rd party plugin can be loaded"
```
Comment (Firstsawyou) [inline .travis/apisix_cli_test/test_main.sh:1039]: ditto.

File: .travis/apisix_cli_test/test_main.sh (hunk 1 above)
Comment (spacewander) [reply .travis/apisix_cli_test/test_main.sh:1036]: Updated.

File: apisix/cli/ops.lua [hunk 3]
```diff
@@ -121,6 +122,19 @@ local function version()
 end
 
 
+local function get_lua_path(conf)
+    if conf then
+        local path = conf
+        if path:byte(-1) ~= str_byte(';') then
+            path = path .. ';'
```
[the remark this answers is not in the document]
Comment (spacewander) [reply apisix/cli/ops.lua:135]: updated.

**Review by membphis — APPROVED** [review]

**Pull request merged**